src/test/java/uk/gov/justice/digital/delius/service/OffenderUpdatesServiceTest.java

```diff
@@ -37,59 +37,59 @@ public void resetRetries() {
 
         @Test
         public void retrievesOffenderDelta() {
```

Comment:
Also need unit tests showing that concurrency errors also happen for `lockNextFailedUpdate`